opencti-platform/opencti-front/src/private/components/cases/incidents/IncidentCreation.tsx

```diff
@@ -129,6 +129,7 @@ export const CaseCreationForm: FunctionComponent<IncidentFormProps> = ({ updater
       description: values.description,
       severity: values.severity,
       priority: values.priority,
+      confidence: parseInt(String(values.confidence), 10),
```

Comment:
What ?
You are parsing a stringify of the value ? Why not use directly the value ?

Reply:
Done, it was an oversight